5_recognition.ipynb

```diff
@@ -60,18 +109,40 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "## 認識対象の配置"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## Place recognition target"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "今回、認識対象とするレゴブロックをHSRの前に出現させましょう。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Enter following command to let Lego block to be appear in front of HSR."
```

Comment:
The following means that it is one of any number of Lego blocks:

    "Enter the following command to make a Lego block to be appear in front of the HSR."

The following means that it is definitely one single previously mentioned/known Lego block:

    "Enter the following command to make the Lego block to be appear in front of the HSR."
￼

Reply:
We only handle single block here, so "a Lego block" is appropriate.

Reply:
Sorry, I just updated the two example sentences from my original comment.
I changed "to be appear" to just "appear".
I missed that... sorry. 

Based on your comment, I think that the sentence you want to use is the following:

   "Enter the following command to make a Lego block appear in front of the HSR."